Update readme & remove acknowledgments (#1007)
handsontable/hyperformula: README.md

<br>
<p align="center">
  <a href="https://handsontable.github.io/hyperformula/">
    <img src="https://raw.githubusercontent.com/handsontable/hyperformula/master/github-hf-logo-blue.svg" width="350" height="71" alt="HyperFormula - A headless spreadsheet, a parser and evaluator of Excel formulas"/>
  </a>
</p>

<p align="center">

>   <strong>A headless spreadsheet, a parser and evaluator of Excel formulas</strong>

Reviewer: Please change the interpunction to:

```
<strong>A headless spreadsheet • A parser and evaluator of Excel formulas</strong>
```

Author: changed

<br>
<p align="center">
  <a href="https://handsontable.github.io/hyperformula/">
    <img src="https://raw.githubusercontent.com/handsontable/hyperformula/master/github-hf-logo-blue.svg" width="350" height="71" alt="HyperFormula - An advanced headless evaluator and parser of the Excel formulas"/>
  </a>
</p>

<p align="center">

>   <strong>An advanced headless evaluator and parser of the Excel formulas</strong>

Reviewer: How about:

```suggestion
  <strong>A headless spreadsheet, an evaluator and parser of Excel formulas</strong>
```

Author: I changed this to:
```
<strong>A headless spreadsheet, a parser and evaluator of Excel formulas</strong>
```

Writing `parser and evaluator` (in this order) makes more sense because you always parse a formula first before evaluating it.

Reviewer: Sure!

  <a href="https://app.fossa.io/projects/git%2Bgithub.com%2Fhandsontable%2Fhyperformula?ref=badge_shield"><img src="https://app.fossa.io/api/projects/git%2Bgithub.com%2Fhandsontable%2Fhyperformula.svg?type=shield" alt="FOSSA Status"></a>
  <a href="https://lgtm.com/projects/g/handsontable/hyperformula/context:javascript"><img src="https://img.shields.io/lgtm/grade/javascript/g/handsontable/hyperformula.svg?logo=lgtm&amp;logoWidth=18" alt="Language grade: JavaScript"></a>
  <a href="https://snyk.io/test/github/handsontable/hyperformula?targetFile=package.json"><img src="https://snyk.io/test/github/handsontable/hyperformula/badge.svg?targetFile=package.json" alt="Known Vulnerabilities"></a>
  <a href="https://github.com/handsontable/hyperformula/graphs/contributors"><img src="https://img.shields.io/github/contributors/handsontable/hyperformula" alt="GitHub contributors"></a>
  <a href="https://codecov.io/gh/handsontable/hyperformula"><img src="https://codecov.io/gh/handsontable/hyperformula/branch/master/graph/badge.svg?token=5k9ZQv8azO" alt="codecov"></a>
</p>

---

> HyperFormula is a headless spreadsheet built on top of TypeScript. It serves as an evaluator and parser for web applications. You can use it in a browser or as a service, with Node.js as your back-end technology.

Reviewer: ```suggestion
HyperFormula is a headless spreadsheet built on top of TypeScript. It serves as an evaluator and parser of Excel formulas for web applications. You can use it in a browser or as a service, with Node.js as your back-end technology.
```

Author: I changed it to:
```
HyperFormula is a headless spreadsheet built on top of TypeScript. It serves as a parser and evaluator of Excel formulas for web applications. You can use it in a browser or as a service, with Node.js as your back-end technology.
```
Writing parser and evaluator (in this order) makes more sense because you always parse a formula first before evaluating it.

Reviewer: @krzysztofspilka agrees

console.log(mySum);
```

[Run this code in CodeSandbox](https://codesandbox.io/s/github/handsontable/hyperformula-demos/tree/develop/basic-usage)

## What can it be used for?

HyperFormula doesn't assume any existing user interface, making it a great general-purpose library that can be used in various business applications. Here are some examples:

- Business logic builders
- Forms and form builders
- Computation notebooks
- Smart documents
- Educational apps
- Online calculators

## Contributing

Help us build the fastest and most flexible calculation engine for
business web apps. Please read the [Contributing Guide](https://handsontable.github.io/hyperformula/guide/contributing.html) before making a pull request.

## License

HyperFormula is available under the open source license ([GPLv3](https://github.com/handsontable/hyperformula/blob/master/gpl-3.0.txt)).

To buy a commercial license, please write to us at sales@handsontable.com

## Copyrights

> © 2021 [Handsoncode](https://handsontable.com)

Reviewer: 2022

Author: fixed
